acceptance/logging_helper.rb

```diff
@@ -75,8 +75,8 @@ def self.run_one_method klass, method_name, reporter
 
 def clean_up_logging_sinks_metrics
   puts "Cleaning up logging sinks and metrics after tests."
-  # $logging.sinks.collect { |s| s.name.starts_with? $prefix }.map &:delete
-  $logging.metrics.collect { |m| m.name.starts_with? $prefix }.map &:delete
+  # $logging.sinks.select { |s| s.name.start_with? $prefix }.map &:delete
+  $logging.metrics.select { |m| m.name.start_with? $prefix }.map &:delete
```

Comment:
@blowmage This is the line I had to modify in order to get the acceptance test cleanup to work for me locally. Previously, `starts_with?`, then `collect` produced errors.